Makefile

```diff
@@ -385,3 +385,26 @@ clean:
 	find . -type f -name 'coverage.html' -delete
 	find . -type f -name 'integration-coverage.txt' -delete
 	find . -type f -name 'integration-coverage.html' -delete
+
+.PHONY: generate-all-labels
+generate-all-labels:
+	$(MAKE) generate-labels TYPE="cmd" COLOR="#483C32"
+	$(MAKE) generate-labels TYPE="pkg" COLOR="#F9DE22"
+	$(MAKE) generate-labels TYPE="extension" COLOR="#FF794D"
+	$(MAKE) generate-labels TYPE="receiver" COLOR="#E91B7B"
+	$(MAKE) generate-labels TYPE="processor" COLOR="#800080"
+	$(MAKE) generate-labels TYPE="exporter" COLOR="#50C878"
+
+.PHONY: generate-labels
+generate-labels:
+	if [ -z $${TYPE+x} ] || [ -z $${COLOR+x} ]; then \
+		echo "Must provide a TYPE and COLOR"; \
+		exit 1; \
+	fi; \
+	echo "Generating labels for $${TYPE}" ; \
+	COMPONENTS=$$(find ./$${TYPE} -type d -maxdepth 1 -mindepth 1 -exec basename \{\} \;); \
+	for comp in $${COMPONENTS}; do \
+		NAME=$${comp//"$${TYPE}"}; \
+		gh label create "$${TYPE}/$${NAME}" -c "$${COLOR}"; \
+	done; \
+	exit 0
```

Comment:
Added this exit because `gh` returns a non-zero exit code if the label already exists and --force is not used.  I didn't want to overwrite any existing labels.